lua/dooing/ui/windows/main.lua

```diff
@@ -0,0 +1,194 @@
+local M = {}
+
+-- Dependencies
+local config = require("dooing.config")
+local state = require("dooing.state")
+
+-- Local window IDs and buffers
+local win_id = nil
+local buf_id = nil
+
+-- Creates and configures the small keys window
+local function create_small_keys_window(main_win_pos)
+	if not config.options.quick_keys then
+		return nil
+	end
+
+	local small_buf = vim.api.nvim_create_buf(false, true)
+	local width = config.options.window.width
+
+	-- Define two separate line arrays for each column
+	local lines_1 = {
+		"",
+		"  i - New todo",
+		"  x - Toggle todo",
+		"  d - Delete todo",
+		"  u - Undo delete",
+		"  H - Add due date",
+		"",
+	}
```

Comment:
I believe keybind-related info should be read from the config itself, to ensure accurate help

Reply:
I should add description in config?

```lua
 keybindings = {
        delete = {
            key = "d",
            description = "Delete todo item"
        },
        undo = {
            key = "u",
            description = "Undo delete"
        },
```

Reply:
Sorry, I wasn't clear. I don't believe descriptions should be in the config. Just showing the actual key defined in the config into help windows👌